src/audio/aria/aria_generic.c

```diff
@@ -83,7 +83,7 @@ void aria_algo_get_data(struct processing_module *mod,
 		for (i = 0; i < n; i += ch_n) {
 			for (ch = 0; ch < ch_n; ch++) {
 				in_sample = *in++;
-				out[ch] = q_multsr_sat_32x32(in_sample, gain, shift);
+				out[ch] = sat_int32(((int64_t)in_sample * gain) >> shift);
```

Comment:
Why is the rounding removed? If really needed, could also add q_mults_sat_32x32() function to format.h. Seems such does not exist.